src/solver/type_getters.jl

```diff
@@ -778,10 +778,16 @@ function get_simulation(config::AtmosConfig)
         spaces = get_spaces(config.parsed_args, params, config.comms_ctx)
     end
 
-    if config.parsed_args["log_params"]
-        filepath = joinpath(sim_info.output_dir, "$(job_id)_parameters.toml")
-        CP.log_parameter_information(config.toml_dict, filepath)
-    end
+    # Check that all set parameters have been used
+    param_filepath =
+        joinpath(sim_info.output_dir, "$(sim_info.job_id)_parameters.toml")
+    CP.log_parameter_information(
+        config.toml_dict,
+        param_filepath,
+        strict = true,
+    )
+    !config.parsed_args["log_params"] && rm(param_filepath)
```

Comment:
```suggestion
    if ClimaComms.iamoot(config.comms_ctx)
      config.parsed_args["log_params"] || rm(param_filepath)
    end
```
Maybe this is the issue? @simonbyrne pointed out that there are currently multiple jobs with the same job id, though.

Reply:
If there are multiple jobs with the same ID this is definitely the problem.

Reply:
Is it ok to rename the job at `config/gpu_configs/gpu_hs_rhoe_equil_55km_nz63_0M_4process.yml`? 

I can also add a test for unique job IDs so that this doesn't happen again.

Reply:
Yeah, I think that's fine. I thought I had added such a test on builkite, actually, but I don't see it. Ah, maybe I'm thinking of `Ensure mse tables are reset when necessary`. Adding such a test would be great.